### PR TITLE
feat(api): add get buckets API

### LINKED ISSUE #36: API: get buckets
### Motivation

Used internally by the CLI.

### Detail

- Method: GET
- Endpoint: /buckets?token&branch
- Response body:
```graphql
[
  {
    id
    name
    commit
    branch
    jobStatus
  }
]
```

### PR DESCRIPTION
**Pull request by neoziro**
Close #36

### CODE REVIEW

File: src/server/routes/api.test.js [hunk 1]
```diff
@@ -27,4 +28,48 @@ describe('api routes', () => {
...
+      });
+    });
+
+    it('should returns buckets', () => {
+      return request(app)
+        .get('/buckets')
+        .set('Host', 'api.argos-ci.dev')
+        .expect(200)
+        .expect((res) => {
+          expect(res.body[0].name).toBe('test-bucket');
```
Comment (oliviertassinari) [inline src/server/routes/api.test.js]: Whould a deep equal be better?

File: src/server/routes/api.js [hunk 2]
```diff
@@ -63,4 +63,16 @@ router.post('/buckets', upload.array('screenshots[]', 50), errorChecking(
   },
 ));
 
+router.get('/buckets', errorChecking(
```
Comment (oliviertassinari) [inline src/server/routes/api.js]: We could have exposed a graphql query. We might need to do the same for the front-end.

**Comment by oliviertassinari** [comment]
Flaky flaky flaky 💥

File: src/server/routes/api.test.js (hunk 1 above)
Comment (neoziro) [inline src/server/routes/api.test.js]: No, I don't want to test the id.

File: src/server/routes/api.js (hunk 2 above)
Comment (neoziro) [inline src/server/routes/api.js]: I think it's not good to mix graphQL and REST, for the CLI I think it's better to be exclusively on REST for now. And I think it's better to separate endpoints.

File: src/server/routes/api.test.js (hunk 1 above)
Comment (oliviertassinari) [inline src/server/routes/api.test.js]: include then?

**Pull request merged**